src/components/visualizations/chart/test/tooltip.spec.ts

```diff
@@ -45,16 +50,17 @@ describe("tooltip", () => {
                 expect(formattedValue).toEqual(" 1");
             },
         );
-        it.each([["0%", 0], ["45.25%", 45.2490089197225], ["100%", 100]])(
-            "should return %s if stackMeasuresToPercent is true and percentageValue is %s",
-            (formattedValue: string, percentageValue: number) => {
-                const tooltip = getFormattedValueForTooltip(
-                    true,
+        it.each([[false, true, 45.24], [false, true, 45.24], [false, true, 45.24]])(
```

Comment:
I think we are losing some test cases here. What would you think about adding expected string value as the first argument in each test run (i.e. `["45.24%", false, true, 45.24], ...`) and add also those existing test cases there: 0, 100 and 45.2490089197225 (they help to test rounding and also some margin values which may be special and useful to test)

Reply:
fixed :)